README.md

```diff
@@ -1,9 +1,18 @@
 # swipe-right-for-cs
 A game for teachers to practice empathizing with young people and authentically connecting their strengths and interests to computer science.
 
-To develop locally:
+This uses a Postgres database; these `psql` commands will boostrap it for local development.
+
+
+### To setup locally:
 ```
 $ yarn install
+$ yarn db-create swipe-right-db
```

Comment:
looks good! script was actually db-create-test but I changed to db-create-dev to keep consistent with the naming convention that we seem to have.

Reply:
hmmmmm. actually. since we made a script to automate the creation of the database and necessary tables, I think we do need a yarn db-create swipe-right-db. just added that to the package.json and the README. we can chat more about this